docs/installation.md

```diff
@@ -174,7 +177,7 @@ The script reads an image named "beatles.jpg" from __data__ folder, runs the det
 Now run the sample script.
 
 ```bash
-(env) $ python3 -m xailient.samples.basic_sample
+(env) $ python3 -m samples/basic_sample.py
```

Comment:
Again, recent changes don't require users to `cd` into the SDK's python package dir. Hence I think we should not make this change.